### PR TITLE
Add travis.yml, add testing samples

### PR DESCRIPTION
**Pull request by iwisunny**
1. fix jest transformer when parsing scss, css, png etc.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -6,12 +6,9 @@
...
     "transform-class-properties",
     ["module-resolver", {
-      "root": ["./"],
+      "root": ["./", "./src/**"],
       "alias": {
-        "core": "./core",
-        "components": "./src/components",
-        "containers": "./src/containers",
-        "utils": "./src/utils"
+        "pages": "./src/containers"
```
Comment (leoendless) [inline .babelrc]: use ``containers`` as contrainer component alias, no need to add ``pages``

File: tests/components/Base/Button.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    const wrapper=shallow(<Button/>);
+  });
+
+  it('primary type', ()=> {
+    const wrapper=shallow(<Button className='primary' />);
+    expect(wrapper.hasClass('primary')).toBeTruthy();
+
+    // since server-side scss is transformed, classnames may not work
+    // const tree=renderer.create(<Button className='primary' />).toJSON();
+    // expect(tree).toMatchSnapshot();
```
Comment (leoendless) [inline tests/components/Base/Button.test.js]: should remove the useless code

File: tests/pages/Home.test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  });
+
+  it('call onEnter', ()=> {
+    // todo
+  });
+
+  // xit('snapshot', ()=> {
+  //   const tree=renderer.create(<Home rootStore={rootStore} />).toJSON();
+  //   expect(tree).toMatchSnapshot();
+  // });
```
Comment (leoendless) [inline tests/pages/Home.test.js]: should remove the useless code

File: .babelrc (hunk 1 above)
Comment (iwisunny) [reply .babelrc]: you can still use `containers/<Any>Component`, add `pages` to alias is optional

File: tests/components/Base/Button.test.js (hunk 2 above)
Comment (iwisunny) [reply tests/components/Base/Button.test.js]: this is just comment, not useless code.

File: tests/pages/Home.test.js (hunk 3 above)
Comment (iwisunny) [reply tests/pages/Home.test.js]: i use `xit` as a skiped test, maybe later will uncomment this section

**Pull request merged**